includedBuild/dependencies/src/main/java/Deps.kt

```diff
@@ -2,7 +2,7 @@ object Versions {
     const val kotlin = "1.4.10"
```

Comment:
To use Deps.kt inside a `plugins` block of root `build.gradle.kts`, `Deps.kt` needs to be placed in `src/main/java`(= no package).